code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -509,4 +532,21 @@
 
 // No binary for pAIs.
 /mob/living/silicon/pai/binarycheck()
-	return 0
\ No newline at end of file
+	return 0
+
+// Handle being picked up.
+
+/mob/living/silicon/pai/get_scooped(var/mob/living/carbon/grabber)

```

Comment:
This has been updated since then, to be considerably less horrible. https://github.com/Zuhayr/Baystation12/blob/dev/code/modules/mob/living/silicon/pai/pai.dm#L420